test/test_docker_api_language_def_json.py

```diff
@@ -146,16 +154,13 @@ def test_docker_build_invalid_lang_def_json(self):
             with open(lang_def_json_path, "w") as f:
                 f.write(json.dumps(lang_def_invalid))
 
-            throwed_exception = False
-            try:
-                build(
-                    flavor_path=(str(temp_flavor_path),),
-                    source_docker_repository_name=self.test_environment.docker_repository_name,
-                    target_docker_repository_name=self.test_environment.docker_repository_name,
-                )
-            except pydantic_core._pydantic_core.ValidationError as e:
-                throwed_exception = True
-            self.assertTrue(throwed_exception)
+            self.assertRaises(
+                pydantic_core._pydantic_core.ValidationError,
```

Comment:
I think, that is not the way you should access the Validation error, because you access a private module 
https://docs.pydantic.dev/latest/errors/errors/

Reply:
Fixed it